src/frameworks/docs/react.md

```diff
@@ -0,0 +1,50 @@
+# Integrate React
+
+Using the Firebase CLI, you can deploy your React static web apps to Firebase 
+and serve them with Firebase Hosting.
+
+Note: Framework-aware Hosting is an early public preview. This means
+that the functionality might change in backward-incompatible ways. A preview
+release is not subject to any SLA or deprecation policy and may receive limited
+or no support.
+
+## Before you begin
+
+Before you get started deploying your app to Firebase,
+review the following requirements and options:
+
+- Firebase CLI version 12.1.0 or later. Make sure to
+  [install the CLI](/docs/cli#install_the_firebase_cli) using your preferred 
+	method.
+- Optional: An existing React project using Vite. You can create one with 
+	`npm create vite@latest -- --template react` or you can let the Firebase CLI
+	to [initialize one](#initialize-a-new-project) for you.
```

Comment:
Suggest "let the Firebase CLI
	 [initialize a new project](#initialize-a-new-project) for you."
	 
	(note the omission of "to")